src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/dummy/DummyPlatform.java

```diff
@@ -75,11 +71,7 @@ public String getName() {
      */
     @Override
     public String getID() {
-        String appendix = "";
-        if (name != null) {
-            appendix = name.toLowerCase();
-        }
-        return ID+appendix;
+        return (ID + " " + name).toLowerCase().replaceAll("[^a-z0-9]", "_");
```

Comment:
We could call it when getting the ID, but that's also in multiple place. We can also add a method to the interface that calls `getId` and applies the regex in the default implementation.